server/src/main/java/org/elasticsearch/transport/InboundPipeline.java

```diff
@@ -134,6 +135,11 @@ private void forwardFragments(TcpChannel channel, ArrayList<Object> fragments) t
         }
     }
 
+    protected void headerReceived(Header header) {
+        assert aggregator.isAggregating() == false;
+        aggregator.headerReceived(header);
+    }
```

Comment:
This is so it can be overriden by the `InboundPipeline` that the `SecurityNetty4Transport` instals for RCS connections.